src/content/platform-integration/ios/app-extensions.md

```diff
@@ -3,62 +3,152 @@ title: Adding iOS app extensions
-involves changing the order of the build process.
-For more information, check out
-[Issue #9690][] and [Issue #135056][].
-:::
+[iOS app extensions]: {{site.apple-dev}}/app-extensions/
+
+## Add an iOS app extension to your Flutter app {: #add-extension }
+
+If you want to integrate your Flutter app with
+the iOS operating system, you can add iOS app extensions
+to your Flutter project. For a seamless workflow, the
+following steps show how to add a `Share Extension`
+app extension to a new Flutter app called
+`example_app_with_extension`, but you can always start with
+an existing project.
 
-[Issue #9690]:   {{site.github}}/flutter/website/issues/9690
-[Issue #135056]: {{site.github}}/flutter/flutter/issues/135056
+1.  In the console, create a new Flutter project called
+    `example_app_with_extension`.
 
-iOS 18 added support for customizing a device's
-Control Center, including creating multiple pages.
-You can also create new toggles for the Control Center
-using the [`ControlCenter`][] API, to feature your app.
+    ```console
+    $ flutter create example_app_with_extension
+    ```
 
-[`ControlCenter`]: {{site.apple-dev}}/documentation/widgetkit/controlcenter
+1.  In the console, open the Xcode workspace for the
+    `example_app_with_extension` project.
 
-## How do you add an app extension to your Flutter app?
+    ```console
+    $ cd example_app_with_extension && open ios/Runner.xcworkspace
+    ```
 
-To add an app extension to your Flutter app,
-add the extension point *target* to your Xcode project.
+1.  In Xcode, add an app extension called `Share Extension`
+    and call it `ShareExtension`.
 
-1. Open the default Xcode workspace in your project by running
-   `open ios/Runner.xcworkspace` in a terminal window from your
-   Flutter project directory.
-1. In Xcode, select **File -> New -> Target** from the menu bar. 
+    *   In the Xcode menu bar, select
+        **File** > **New** > **Target**.
+    
+    *   Add **Share Extension**.
 
-    <figure>
-      <div class="site-figure-container">
-        <img src='/assets/images/docs/development/platform-integration/app-extensions/xcode-new-target.png' alt='Opening the File -> New menu, then selecting Target in Xcode.' height='300'>
-      </div>
-    </figure>
-1. Select the app extension you intend to add.
-   This selection generates extension-specific code 
-   within a new folder in your project.
-   To learn more about the generated code and the SDKs for each
-   extension point, check out the resources in
-   [Apple's documentation][].
+    *   In the **Name field**, enter **ShareExtension**.
 
-To learn how to add a home screen widget to your iOS device,
-check out the 
-[Adding a Home Screen Widget to your Flutter app][lab]
-codelab.
+    *   Click **Finish**.
+
+    *   In the **Activate … Scheme** dialog box that
+        appears, select **Activate**.
+
+1.  In Xcode, change the order of the build process.
+
+    *   In the **project navigator**, at the top, select
```

Comment:
In other place you give these instructions too:
```
Open the project navigator (View > Navigators > Project)
```

Reply:
Updated here and elsewhere.